CRFModuleValidation/CRFMotionRecorder.swift

```diff
@@ -37,10 +37,14 @@ import ResearchSuite
 
 public enum CRFMotionRecorderType : String, Codable {
     case accelerometer
```

Comment:
would someone ever maybe want the raw gyroscope data? They can get rotationRate from the device motion, but that's been processed, which is probably usually more useful but...? There's no existing use case, but it may at least be worth mentioning in an issue on Github.